api/BasicCardRequest.json

```diff
@@ -18,6 +18,7 @@
           },
           "firefox": {
             "version_added": "56",
+            "notes": "Available only in nightly builds.",
```

Comment:
We might need to also add the pref:

```
dom.payments.request.supportedRegions
```

Which takes a list of ISO country codes, like `US,CA`.